src/dal/Jalasoft.TeamUp.Projects.DAL/ProjectsInMemoryRepository.cs

```diff
@@ -74,5 +74,14 @@ public Project Add(Project project)
             projects = new List<Project>(projects) { project }.ToArray();
             return project;
         }
+
+        public Project DeleteById(Guid id)
+        {
+            var projectsList = new List<Project>(projects);
```

Comment:
is this auxiliary variable needed? can't we use directly the global one?

Reply:
Done, using the global variable